beps/0001-notifications-system/README.md

```diff
@@ -427,22 +427,22 @@ The notification and signal plugins are released as two new plugins in the Backs
 
 For the notification plugin to reach a stable release we much reach the following:
 
-- A stable notifications payload format.
-- A stable notifications recipient filter format.
-- The event broker must have at least one implementation that supports scaled deployments.
+- [ ] A stable notifications payload format.
+- [ ] A stable notifications recipient filter format.
+- [x] The events service must have at least one implementation that supports scaled deployments. Done in #24916.
 
 For the signal plugin to reach a stable release we much reach the following:
 
-- A stable signal recipient filter format.
-- A stable signal channel API in the frontend.
+- [ ] A stable signal recipient filter format.
+- [ ] A stable signal channel API in the frontend.
 
 If any changes are required to the frontend framework to facilitate the implementation of notifications or signals, these will be released as experimental alpha features. They will stay in alpha until they are deemed stable enough, which must happen before a stable release of the notifications system.
 
 ## Dependencies
 
-Since the signal plugin relies on the event broker for communication, it is a dependency for the notifications system as a whole. The event broker does not currently implement any transport for scaled deployments, which is a requirement for scaled deployments of the notification system.
+~Since the signal plugin relies on the events service for communication, it is a dependency for the notifications system as a whole. The events service does not currently implement any transport for scaled deployments, which is a requirement for scaled deployments of the notification system.
```

Comment:
```suggestion
Since the signal plugin relies on the events service for communication, it is a dependency for the notifications system as a whole. The events service does not currently implement any transport for scaled deployments, which is a requirement for scaled deployments of the notification system.
```